src/bilder/images/airbyte/templates/.env.tmpl

```diff
@@ -0,0 +1,100 @@
+# originally taken from: https://github.com/airbytehq/airbyte
+#
+# This file only contains Docker relevant variables.
+#
+# Variables with defaults have been omitted to avoid duplication of defaults.
+# The only exception to the non-default rule are env vars related to scaling.
+#
+# See https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/java/io/airbyte/config/Configs.java
+# for the latest environment variables.
+#
+# # Contributors - please organise this env file according to the above linked file.
+
+### SHARED ###
+VERSION=0.39.8-alpha
+
+# When using the airbyte-db via default docker image
+CONFIG_ROOT=/data
+DATA_DOCKER_MOUNT=airbyte_data
+DB_DOCKER_MOUNT=airbyte_db
+
+# Workspace storage for running jobs (logs, etc)
+WORKSPACE_ROOT=/tmp/workspace
+WORKSPACE_DOCKER_MOUNT=airbyte_workspace
+
+# Local mount to access local files from filesystem
+# todo (cgardens) - when we are mount raw directories instead of named volumes, *_DOCKER_MOUNT must
+# be the same as *_ROOT.
+# Issue: https://github.com/airbytehq/airbyte/issues/578
+LOCAL_ROOT=/tmp/airbyte_local
+LOCAL_DOCKER_MOUNT=/tmp/airbyte_local
+# todo (cgardens) - hack to handle behavior change in docker compose. *_PARENT directories MUST
+# already exist on the host filesystem and MUST be parents of *_ROOT.
+# Issue: https://github.com/airbytehq/airbyte/issues/577
+HACK_LOCAL_ROOT_PARENT=/tmp
+
+
+JOBS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION=0.29.15.001
+
+# Airbyte Internal Config Database, defaults to Job Database if empty. Explicitly left empty to mute docker compose warnings.
+CONFIG_DATABASE_USER=
+CONFIG_DATABASE_PASSWORD=
+CONFIG_DATABASE_URL=
+CONFIGS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION=0.35.15.001
+
+### AIRBYTE SERVICES ###
+TEMPORAL_HOST=airbyte-temporal:7233
+INTERNAL_API_HOST=airbyte-server:8001
+WEBAPP_URL=http://localhost:8000/
+# Although not present as an env var, required for webapp configuration.
+API_URL=/api/v1/
+
+
+### JOBS ###
+# Relevant to scaling.
+SYNC_JOB_MAX_ATTEMPTS=3
+SYNC_JOB_MAX_TIMEOUT_DAYS=3
+JOB_MAIN_CONTAINER_CPU_REQUEST=
+JOB_MAIN_CONTAINER_CPU_LIMIT=
+JOB_MAIN_CONTAINER_MEMORY_REQUEST=
+JOB_MAIN_CONTAINER_MEMORY_LIMIT=
+
+
+### LOGGING/MONITORING/TRACKING ###
+TRACKING_STRATEGY=segment
+# Although not present as an env var, expected by Log4J configuration.
+LOG_LEVEL=INFO
+# Although not present as an env var, helps Airbyte track job healthiness.
+SENTRY_DSN="https://d4b03de0c4574c78999b8d58e55243dc@o1009025.ingest.sentry.io/6102835"
```

Comment:
Is this a valid Sentry DSN for us, or is this something from the upstream config? Might be worth using our own Sentry and moving the DSN into Vault config so it doesn't get spammed.

Reply:
I honestly didn't even notice it. We should replace it with our own, I agree.

Reply:
I set this up but it doesn't seem to feed any data into the project. Prob not worth spending much time on at this moment.